### PR TITLE
Abhinav localizer cals

### PR DESCRIPTION
**Pull request by abhinavGirish**

### CODE REVIEW

File: Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env/python
```
Comment (VivaanBahl) [inline Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py]: What is this file for?

File: Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp [hunk 2]
```diff
@@ -123,9 +123,19 @@ void Localizer::init_R()
     ;
 
     std::stringstream s;
+    XmlRpc::XmlRpcValue v;
```
Comment (VivaanBahl) [inline Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp]: Is there a way for this to just be an array of doubles? Does it have to be this specific class?

File: Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py (hunk 1 above)
Comment (abhinavGirish) [reply Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py]: that was just meant to be for learning about ROS along with the rest of the team; I put it on my own branch but forgot to take it out

File: Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp (hunk 2 above)
Comment (abhinavGirish) [reply Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp]: I'm not totally sure, will have to look into it more - that's just what I tried after looking up a ROS forum on reading values from a launch file

Comment (VivaanBahl) [reply Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp]: Check out the ros docs: http://wiki.ros.org/roscpp/Overview/Parameter%20Server#Retrieving_Lists
We have kinetic (newer than groovy), so we don't need to use this method

Comment (abhinavGirish) [reply Software/real_time/ROS_RoboBuggy/src/robobuggy/src/transistor/localizer/Localizer.cpp]: resolved - check again?

File: Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py (hunk 1 above)
Comment (abhinavGirish) [reply Software/real_time/ROS_RoboBuggy/src/robobuggy/scripts/basic_gps_subscriber.py]: resolved

**Pull request merged**